README.md

```diff
@@ -1,86 +1,257 @@
+<img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">
 
-<sub>Remember to star it, flow will love you more :)</sub>
 
----
+## 🎉 New Features in 1.9 
+
+![screenshot](https://user-images.githubusercontent.com/6903107/144855345-45535bc7-7777-4c5a-b8d9-d6ce8adc5e84.png)
+
+- All New Design. New Themes, New Setting Window. Animation & Sound Effect, Color Scheme aka Dark Mode.
+- New Plugins, Plugin Store, Game Mode, Wizard window
+- <a href="https://github.com/Flow-Launcher/Flow.Launcher/releases/tag/v1.9.0">Full changelog</a>
+
+<img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">
 
 <h4 align="center">
-  <a href="#Features">Features</a> •
-  <a href="#Getting-Started">Getting Started</a> •
+  <a href="#Getting-Started">Getting Started</a> •  <a href="#Features">Features</a> • <a href="#Plugins">Plugins</a> •
+       <a href="#Hotkeys">Hotkeys</a> •
   <a href="#QuestionsSuggestions">Questions/Suggestions</a> •
   <a href="#Development">Development</a> •
-  <a href="https://flow-launcher.github.io/docs">Documentation</a>
+  <a href="https://flow-launcher.github.io/docs">Docs</a>
 </h4>
 
----
+<img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">
 
-## Features
+## 🚗 Getting Started
 
-![The Flow](https://user-images.githubusercontent.com/26427004/82151677-fa9c7100-989f-11ea-9143-81de60aaf07d.gif)
+### Installation
+
+| [Windows 7+ installer](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest/download/Flow-Launcher-Setup.exe) | [Portable](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest/download/Flow-Launcher-Portable.zip) | `WinGet install "Flow Launcher"` |
+| :----------------------------------------------------------: | :----------------------------------------------------------: | :------------------------------: |
+
+> Windows may complain about security due to code not being signed, this will be completed at a later stage. If you downloaded from this repo, you are good to continue the set up. 
+
+And You can download <a href="https://github.com/Flow-Launcher/Flow.Launcher/discussions" target="_blank">early access version</a>.
+
+<img src="https://user-images.githubusercontent.com/6903107/144858082-8b654daf-60fb-4ee6-89b2-6183b73510d1.png" width="100%">
+
+## 🎁 Features
+
+### Applications & Files
+
+<img src="https://user-images.githubusercontent.com/6903107/144517107-efd83586-9084-46e0-b7dc-79a6748ea936.png" width="500">
+
+
+- Search for files or their contents.
+
+<img src="https://user-images.githubusercontent.com/6903107/144862511-bb5a28c6-6aa8-4298-9b63-9306d5e8dcda.png" width="400">
```

Comment:
just realised this does not show environment variables search, may be do one with this query `%`

Reply:
I don't get it. Do you mean it's better to change it to this screen?

![image](https://user-images.githubusercontent.com/6903107/145016131-acdcc710-c2da-4c30-9218-b60f517710bc.png)